nuxt.config.js

```diff
@@ -322,6 +322,16 @@ export default defineNuxtConfig({
   },
 
   hooks: {
+    ready(nuxt) {
+      // https://github.com/nuxt/bridge/issues/607
+      // translate nuxt 2 hook from @nuxt/webpack-edge to nuxt bridge hook
+      nuxt.hook('server:devMiddleware', async (devMiddleware) => {
```

Comment:
let's remove this file https://github.com/kodadot/nft-gallery/blob/main/patches/%40nuxt__webpack-edge%402.16.0-27720022.54e852f.patch